go/client/cmd_team_add_members_bulk.go

```diff
@@ -129,7 +129,8 @@ func (c *CmdTeamAddMembersBulk) Run() error {
 	}
 	c.arg.TeamID = teamID
 
-	err = cli.TeamAddMembersMultiRole(context.Background(), c.arg)
+	// currently ignoring res
+	_, err = cli.TeamAddMembersMultiRole(context.Background(), c.arg)
```

Comment:
is this the desired behavior?

Reply:
it's nice to print the results out to cli which is used for debugging a lot

Reply:
ok. Run() can only return an error, so i'll create a new CLI error that will be returned when some but not all members are added

Reply:
will address in PICNIC-714